### PR TITLE
Fix/unclickable enrollment button

### LINKED ISSUE #110: Enrollment button doesn't work
This is a really weird one. From my own testing it seems like the browser waits a little bit before following the link and then 'can't find it anymore', because it was moved by the animation. Could probably be hotfixed by triggering the link from the OnTouchEnd event or putting a invisible `<a>` over everything while the card is transitioning out. I would really like to know why this is happening though. Any thought @emielvanrijn @Phavion ?

### PR DESCRIPTION
**Pull request by cdfa**
Should fix #110

### CODE REVIEW

**Comment by emielvanrijn** [comment]
On mobile this is still a bit quirky. Somehow, it does not always respond immediatly to touches?

**Comment by cdfa** [comment]
I thought it was pretty responsive when I tried it. Can you send a video?

**Comment by emielvanrijn** [comment]
Nevermind, I think this was just an issue with my phone :)... 
LGTM!

File: src/components/activities/Activity.jsx [hunk 1]
```diff
@@ -136,9 +107,11 @@ const FullSizeCard = styled(Card)`
 `;
 
 const TurnRevealWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+
   .turnreveal-container {
-    display: flex;
-    justify-content: center;
+    width: fit-content;
```
Comment (emielvanrijn) [inline src/components/activities/Activity.jsx]: Such nice stuff :).

**Pull request merged**